website/docs/api/misc/eslint-plugin/no-untranslated-text.md

```diff
@@ -0,0 +1,48 @@
+---
+slug: '/api/misc/@docusaurus/eslint-plugin/no-untranslated-text'
+---
+
+enforce text labels in JSX to be wrapped by translate calls (no-untranslated-text)
```

Comment:
Need to be an h1 title

Reply:
Hmm could it be an h2 ? The title "string-literal-i18n-messages" is h1 too, so there will be two h1s in a row. Also when I make it an h1, the title disappears. I'm sending two screenshots to demonstrate, one with h1 and the other with h2.

![h1](https://user-images.githubusercontent.com/18152197/164638409-9e10cd35-0b5b-4dbb-8fda-2a9f3b2d567c.png)
![h2](https://user-images.githubusercontent.com/18152197/164638468-5b781c3d-d388-4035-b866-c2e62435ba0d.png)

Reply:
Fine, it seems the only difference is that it won't appear in table of contents. The size also seems to be adjusted to look like an h2, I'll make it an h1.

Reply:
Yes, FYI there're discussions in both ESLint and TS-ESLint docs about whether the rule description should be in the title. Both seem to have it removed now (https://github.com/typescript-eslint/typescript-eslint/pull/4367, https://github.com/eslint/eslint/pull/15514)

Reply:
Addressd in 492615f.

Reply:
Can we _not_ have the description as an h1 heading as well? It's bad practice to have 2 h1 headings. TS-ESLint and ESLint both have the rule description as a normal paragraph.